### PR TITLE
Allow specification of custom service account name

### LINKED ISSUE #107: Allow customizing service account
Allow to re-use a specific existing service acount

### PR DESCRIPTION
**Pull request by tuimz**
And hide creation of service account behind a boolean (defaults to true to maintain compatibility)

# Pull Request

## What does this PR do?
Fixes #107 

## PR checklist
Please check if your PR fulfills the following requirements:
- [x] Does this PR fix an existing issue?
- [x] Have you read the contributing guidelines?
- [x] Have you made sure that the title is accurate and descriptive of the changes?

Thank you so much for contributing to Meilisearch!

### CODE REVIEW

File: charts/meilisearch/values.yaml [hunk 1]
```diff
@@ -35,6 +35,10 @@ auth:
   existingMasterKeySecret: ""
 
 serviceAccount:
+  # You can use a specific pre-created service account instead of
+  # the one created by this chart. Set create to false and enter a name
+  create: true
```
Comment (alallema) [inline charts/meilisearch/values.yaml:40]: Is not the opposite false you use the default account and true to create one?

**Review by alallema — COMMENTED** [review]
Thank you so much @tuimz for this PR 🔥 
I just add a question. You also need to update the chart version.

File: charts/meilisearch/values.yaml (hunk 1 above)
Comment (tuimz) [reply charts/meilisearch/values.yaml:41]: Cheers, I have clarified the variable.

`create: false` prevents this chart from creating a service account.

**Review by alallema — APPROVED** [review]
LGTM!
Thanks @tuimz for contributing to Meilisearch ❤️

**Comment by alallema** [comment]
bors merge

**Pull request merged**